### ISSUE TITLE
Fieldset Field Label "for" does not match Input "id"

### ISSUE THREAD

**Issue by lukearmstrong** [fuel/core#831]
I've recently updated `fuel/core` on an existing site, and I've just noticed that when clicking the labels, the inputs aren't focused because the label "for" does not match the input "id". This is rather annoying for checkboxes/radio buttons.

``` php
<?= $fieldset->field('name'); ?>
```

**Generates this:**

``` html
<div class="clearfix ">
    <label for="form_name">Name</label>
    <div class="input">
        <input type="text" required="required" value="Adam" id="name" name="name" /><span class="help-inline">*</span>
    </div>
</div>
```

**When it used to, and should, generate this:**

``` html
<div class="clearfix ">
    <label for="form_name">Name</label>
    <div class="input">
        <input type="text" required="required" value="Adam" id="form_name" name="name" /><span class="help-inline">*</span>
    </div>
</div>
```

**Comment by jschreuder** [comment]
Closed, will be handled by issue #839.

**Comment by jschreuder** [comment]
This should be fixed now: aa197e81bc0fe6ebaaf732747ce6160d3500229f
